wiki/development/coding-guidelines.md

```diff
@@ -380,18 +382,18 @@ If a function returns error information, then that error information will be tes
 There shall be no unreachable code.
 
 ### 6.3. Function Parameters
-Parameters of functions must be retrieved through the usage of the param or params commands. If the function is part of the public API, parameters must be checked on allowed data types and values through the usage of the param and params commands.
+Parameters of functions must be retrieved through the usage of the `param` or `params` commands. If the function is part of the public API, parameters must be checked on allowed data types and values through the usage of the param and params commands.
 
 Usage of the CBA Macro `PARAM_x` or `BIS_fnc_Param` is deprecated and not allowed within the ACE project unless very specific reasons allow no other alternative.
 
 ### 6.4. Return Values
-Functions and code blocks that specific a return a value must have a meaningfull return value.
+Functions and code blocks that specific a return a value must have a meaningfull return value. If no meaningfull return value, the function should return nil.
 
 ### 6.5. Private Variables
-All private variables shall be declared in a private array.
+All private variables shall make use of the `private` keyword on initalization. When declaring a private variable before initalization, usage of the private array syntax is allowed. All private variables must be either initialized using the private key word, or declared using the private array syntax.
```

Comment:
Or by using `params` I guess.
`params` is not exclusive to the functions arguments, but is sometimes also used in forEach loops (`_x params [...]`), so it might be worth noting.